src/erlcloud_kms.erl

```diff
@@ -986,10 +986,19 @@ kms_result_fun(#aws_request{response_type = error, error_type = aws} = Request)
     Request#aws_request{should_retry = false}.
 
 
-%
-% Private
-%
-request(Config, Operation, Body) ->
+%%%------------------------------------------------------------------------------
+%%% Internal Functions
+%%%------------------------------------------------------------------------------
+
+kms_request(Config, Operation, Body) ->
+    case erlcloud_aws:update_config(Config) of
```

Comment:
I just replicated the way other `erlcloud_*` integrations are implemented to use iam roles, see https://github.com/alertlogic/erlcloud/blob/11007e2c522c7e0751cea50020e2f79850615f91/src/erlcloud_ses.erl#L958